evaluations.css

```diff
@@ -0,0 +1,97 @@
+		body {
+			min-width: 100%;
+			position: relative;
+			font-family: Arial;
+			background-color: #ffffff; 
+			margin: auto;
+		}
+
+		header {
+			height: 70px;
+			border-bottom: 1px solid grey;
+		}
+
+		footer {
+			height: 60px; 
+			position: absolute;
+			left: 0;
+			right: 0;
+			line-height: 60px;
+			background-color: #bcbcaf;
+			color: #686865;
+			font-family: Arial;
+			text-align: right;
+			padding-right: 50px;
+		
+		}
+
+		a {
+			vertical-align: middle;
+			line-height: 70px;
+			margin-left: 100px;
```

Comment:
I'd have to look at the whole context more closely. I'm not saying I have an issue with how it's done here, I just found it a bit strange that it's such a considerably large value, at first glance.